api/v1beta1/nodemaintenance_webhook_test.go

```diff
@@ -196,7 +196,7 @@ func getTestNode(name string, isControlPlane bool) *v1.Node {
 	}
 	if isControlPlane {
 		node.ObjectMeta.Labels = map[string]string{
-			LabelNameRoleControlPlane: "",
+			"node-role.kubernetes.io/control-plane": "",
```

Comment:
NIT: We can just use it from https://github.com/medik8s/common/blob/main/pkg/labels/labels.go#L9

Reply:
Not something I feel strongly about in this case, but the idea is to decouple test code from production code.